### PR TITLE
GW Entity Targets

### PR DESCRIPTION
**Pull request by Guaris**
### Preview Links

Pulled from here
https://github.com/Kong/developer.konghq.com/pull/168

### Checklist 

- [ ] Every page is page one
- [ ] Tested how-to docs. If not, note why here. 
- [ ] All pages contain metadata
- [ ] Updated [sources.yaml](https://github.com/Kong/developer.konghq.com/blob/main/tools/track-docs-changes/config/sources.yml). For more info review [track docs changes](https://github.com/Kong/developer.konghq.com/tree/main/tools/track-docs-changes)
- [ ] Any new docs link to existing docs.

### CODE REVIEW

File: app/_gateway_entities/target.md [hunk 1]
```diff
@@ -11,22 +11,65 @@ tools:
     - deck
     - terraform
```
Comment (fabianrbz) [inline app/_gateway_entities/target.md:13]:
```suggestion
entities:
  - target

```

File: app/_gateway_entities/target.md [hunk 2]
```diff
@@ -11,22 +11,65 @@ tools:
...
+<!--vale on-->
 
 ## Schema
 
 {% entity_schema %}
+
+## Set up a Target
+
+{% entity_example %}
+type: target
```
Comment (fabianrbz) [inline app/_gateway_entities/target.md:69]:
```suggestion
type: target
data:
  target: httpbun.com:80
  weight: 100

```

This should be just one target

File: app/_gateway_entities/target.md [hunk 3]
```diff
@@ -11,22 +11,65 @@ tools:
...
+data:
+   - target: httpbun.com:80
+     weight: 100
+   - target: httpbin.konghq.com:80
+     weight: 100
```
Comment (fabianrbz) [inline app/_gateway_entities/target.md:74]: we need to add the instructions [here](https://github.com/Kong/developer.konghq.com/blob/main/app/_includes/components/entity_example/format/ui.md?plain=1#L44)

File: app/_gateway_entities/target.md [hunk 4]
```diff
@@ -11,22 +11,65 @@ tools:
...
+  end
 
-## Use cases for targets
+  subgraph id2 ["`Targets (example_upstream)`"]
+    C --> D & E
+  end
 
-The following are examples of common use cases for targets:
+  style id1 rx:10,ry:10
+  style id2 stroke:none
```
Comment (fabianrbz) [inline app/_gateway_entities/target.md:57]: we can remove these lines

File: app/_gateway_entities/target.md [hunk 5]
```diff
@@ -11,22 +11,65 @@ tools:
...
+  C(Load balancer)
+  D(httpbin.konghq.com)
+  E(httpbun.com)
+  
+  subgraph id1 ["`**KONG GATEWAY**`"]
+    A --> B --> C
+  end
 
-## Use cases for targets
+  subgraph id2 ["`Targets (example_upstream)`"]
```
Comment (fabianrbz) [inline app/_gateway_entities/target.md:52]:
```suggestion
  subgraph id2 ["`Targets`"]
```

This is a known [issue](https://github.com/mermaid-js/mermaid/issues/3806), I would say let's remove the extra text for now

File: app/_gateway_entities/target.md [hunk 6]
```diff
@@ -11,22 +11,39 @@ tools:
...
-description: A target identifies a specific instance of an upstream service.
+description: A Target is an IP address/hostname with a port that identifies an instance of a backend service.
+
+related_resources:
+  - text: Upstream entity
+    url: /gateway/entities/upstream/
+  - text: Routing in {{site.base_gateway}}
+    url: /gateway/routing/
+  - text: Proxying with {{site.base_gateway}}
+    url: /gateway/traffic-control/proxy/
```
Comment (cloudjumpercat) [inline app/_gateway_entities/target.md:22]:
```suggestion
```

File: app/_gateway_entities/target.md [hunk 7]
```diff
@@ -11,22 +11,39 @@ tools:
...
     - terraform
 
-description: A target identifies a specific instance of an upstream service.
+description: A Target is an IP address/hostname with a port that identifies an instance of a backend service.
+
+related_resources:
+  - text: Upstream entity
+    url: /gateway/entities/upstream/
+  - text: Routing in {{site.base_gateway}}
+    url: /gateway/routing/
```
Comment (cloudjumpercat) [inline app/_gateway_entities/target.md:20]:
```suggestion
  - text: Route entity
    url: /gateway/entities/route/
```

**Review by cloudjumpercat — APPROVED** [review]

**Pull request merged**